types/wicket/index.d.ts

```diff
@@ -1,46 +1,48 @@
-declare var Wkt: WktModule.WktStatic;
+export {};
```

Comment:
`wicket` uses UMD, per https://github.com/arthur-e/Wicket/blob/5f9162f8ad98cc044bd00a7d925caa84b72397f7/wicket.js#L22 so you should also have an `export as namespace` declaration.

```suggestion
export {};
export as namespace Wkt;
```